doc/building_blocks/data_preparation.rst

```diff
@@ -99,13 +101,14 @@ a look at it.
     :start-after: # Simple mask extraction from EPI images
     :end-before: # Generate mask with strong opening
 
-____
 
-.. figure:: ../auto_examples/manipulating_visualizing/images/plot_mask_computation_3.png
+.. figure:: ../auto_examples/manipulating_visualizing/images/plot_mask_computation_2.png
     :target: ../auto_examples/plot_mask_computation.html
-    :align: right
+    :align: left
     :scale: 50%
 
+____
```

Comment:
Yes. Since images aren't sideimages anymore, we should make clear connection between a code snippet and its output.